packages/flutter_tools/test/general.shard/license_collector_test.dart

```diff
@@ -37,50 +39,40 @@ const String _kApacheLicense = r'''
                                  Apache License
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
-
```

Comment:
Did you mean to remove the whitespace?

Reply:
I think my IDE autoformatted this 🤦‍♂

Reply:
Ahh, I configured Git wrong for windows or something.